### PR TITLE
Handle no child folder/document in folder view

### PR DESCRIPTION
**Pull request by vidhibagadia**
Handles Issue #7 

This PR adds a No Object [Folder/Document] for the folder view.
Future Scope: Style the messages as per the UI of the entire app.

### CODE REVIEW

File: apps/docweb/templates/docweb/repository.html [hunk 1]
```diff
@@ -144,6 +144,9 @@ <h2>Repository</h2>
         </div>
         <div class="children">
```
Comment (hseritt) [inline apps/docweb/templates/docweb/repository.html]: Hi! Appreciate your help! This is good and works but have you considered maybe using something like:

```
{% for ... %}
    ...
{% empty %}
    ...
{% endfor %}
```

See: https://docs.djangoproject.com/en/3.0/ref/templates/builtins/#for-empty

Comment (vidhibagadia) [reply apps/docweb/templates/docweb/repository.html]: Hi, thanks! I was unaware of the `for..empty` tag. Looks like a really cleaner approach for the same thing, made the changes. :)

**Comment by vidhibagadia** [comment]
Just noticed, the folders' list has `table` being looped, while for documents, the `tr` is getting looped. Is this intentional?

**Comment by hseritt** [comment]
I believe that's a bug. The table should not be looped. I think I fixed
that in a newer version. But I'm not 100% sure.

**Comment by hseritt** [comment]
Thanks!

**Comment by hseritt** [comment]
Looks good! I made a few changes (due to my mistake with including table in the for loop). I'll be merging this to the project's master. Whenever you like, you should be able to do a git pull on master and see your changes included. Thanks much! Feel free to grab another issue and work it if you like :-)

**Pull request merged**